packages/react/src/primitives/TextField/__tests__/TextField.test.tsx

```diff
@@ -185,136 +175,6 @@ describe('TextField component', () => {
     });
   });
 
-  describe('Multiline textarea field', () => {
```

Comment:
Should these tests be included in `TextAreaField.test.tsx`?

Reply:
They [already were](https://github.com/aws-amplify/amplify-ui/blob/main/packages/react/src/primitives/TextAreaField/__tests__/TextAreaField.test.tsx#L59) when I created the TextAreaField. At the time I left these here while we waiting for customers to transition away from the `isMultiline` prop to the separate TextAreaFieldcomponent. Now they can be fully removed. 😃